ops/terraform/env/mgmt/github-actions-iam.tf

```diff
@@ -514,7 +514,8 @@ resource "aws_iam_openid_connect_provider" "github_actions" {
 
 resource "aws_iam_role" "github_actions" {
   name        = "bfd-${local.env}-github-actions"
-  path        = "/"
+  path        = local.cloudtamer_iam_path
+#  permissions_boundary = data.aws_iam_policy.permissions_boundary.arn # Commented out until an official migration due to incompatabilities
```

Comment:
Remove this. Comment above the resource with a defined story if you think it would be helpful to remember in the Greenfield migration.
```suggestion
```